app/uk/gov/hmrc/channelpreferences/controllers/PreferenceController.scala

```diff
@@ -76,4 +83,100 @@ class PreferenceController @Inject()(
           s", groupId: '${event.groupId}', timeStamp: '${event.timeStamp}', event: '${event.event}'"))
     }
   }
+
+  // ----------------------
+
+  private def doConfirmItsa(passedBack_entityId: String, passedBack_itsaId: String, authToken_saUtr: Option[String])(
```

Comment:
Are there any reason to have `snake_case` on the methods parameters?
`passedBack_entityId`, `passedBack_itsaId`, etc...

Reply:
No reasons in particular

Reply:
we try to avoid snake casing

Reply:
Ok, got it.
You like `camelCase` instead of `snake_case` (that looks more Scala-ish ... agreed!)